Client/Frontend/Shields/ShieldsViewController.swift

```diff
@@ -133,6 +137,7 @@ class ShieldsViewController: UIViewController, PopoverContentComponent {
         (.AllOff, shieldsView.shieldOverrideControl, nil),
         (.AdblockAndTp, shieldsView.shieldsContainerStackView.adsTrackersControl, Preferences.Shields.blockAdsAndTracking),
         (.SafeBrowsing, shieldsView.shieldsContainerStackView.blockMalwareControl, Preferences.Shields.blockPhishingAndMalware),
+//        (.GoogleSafeBrowsing, shieldsView.shieldsContainerStackView.googleSafeBrowsingControl, Preferences.Shields.googleSafeBrowsing)
```

Comment:
Should this be commented-out?

Reply:
Yes it should. It's no longer under its own toggle. It's under the `blockMalwareControl` toggle as requested in Slack.

Reply:
Ok, so I suppose we can just remove the line now?